client/ifdown.c

```diff
@@ -106,13 +106,19 @@ ni_ifdown_fire_nanny(ni_ifworker_array_t *array)
 int
 ni_do_ifdown(int argc, char **argv)
 {
-	enum  { OPT_HELP, OPT_FORCE, OPT_DELETE, OPT_NO_DELETE, OPT_TIMEOUT };
+	enum  {
+		OPT_DELETE	= 'd',
+		OPT_FORCE	= 'f',
+		OPT_HELP	= 'h',
+		OPT_NO_DELETE	= 'n',
+		OPT_TIMEOUT	= 't'
+	};
 	static struct option ifdown_options[] = {
-		{ "help",	no_argument, NULL,		OPT_HELP },
-		{ "force",	required_argument, NULL,	OPT_FORCE },
-		{ "delete",	no_argument, NULL,	OPT_DELETE },
-		{ "no-delete",	no_argument, NULL,	OPT_NO_DELETE },
-		{ "timeout",	required_argument, NULL,	OPT_TIMEOUT },
+		{ "help",	no_argument,		NULL,	OPT_HELP	},
+		{ "force",	required_argument,	NULL,	OPT_FORCE	},
+		{ "delete",	no_argument,		NULL,	OPT_DELETE	},
+		{ "no-delete",	no_argument,		NULL,	OPT_NO_DELETE	},
```

Comment:
I guess it will behave as `wicked ifdown --delete --no-delete`? It sure is a weird combination but I didn't add anything new there, just added shortcuts, or am I missing something?